plugin-api/build.gradle

```diff
@@ -53,7 +53,6 @@ shadowJar {
     exclude(project(':check-api'))
   }
   relocate('com.google', 'org.sonar.api.internal.google')
-  relocate('org.apache.commons', 'org.sonar.api.internal.apachecommons')
```

Comment:
Not sure why that was needed, but it shouldn't be needed for commons-lang3.

Reply:
This is to make it clear that plugins should not rely on the classes transitively. This is implementation details.

Reply:
So I think we should keep relocation

Reply:
Ok, I put it back 👍